test/e2e/config/e2e_conf.yaml

```diff
@@ -70,6 +70,8 @@ providers:
       targetName: "cluster-template-ha.yaml"
     - sourcePath: "${PWD}/test/e2e/_out/cluster-template-prow-ha-centos.yaml"
       targetName: "cluster-template-ha-centos.yaml"
+    - sourcePath: "${PWD}/test/e2e/_out/cluster-template-prow-ha-m3-dev-env.yaml"
+      targetName: "cluster-template-ha-m3-dev-env.yaml"
```

Comment:
Are `cluster-template-ha-centos.yaml` and `cluster-template-ha.yaml` templates used anywhere since now flavorSuffix is same?

Reply:
Let's comment them out and add above points so we can make use of them as a reference/whole in the future.

Reply:
Fixed! Thanks.